apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -37,6 +37,15 @@ class YourSchool extends Component {
     alertHeading: PropTypes.string,
     alertText: PropTypes.string,
     alertUrl: PropTypes.string,
+    userName: PropTypes.string,
+    userEmail: PropTypes.string,
+    isTeacher: PropTypes.bool,
+    schoolCountry: PropTypes.string,
+    schoolId: PropTypes.string,
+    schoolType: PropTypes.string,
+    schoolName: PropTypes.string,
+    schoolState: PropTypes.string,
+    schoolZip: PropTypes.string,
```

Comment:
Can we share this prop validation with the list in `CensusForm`?

Reply:
Done.